src/users/queries/handlers/get-users.handler.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { UsersRepository } from '../../repositories/users-repository';
```

Comment:
Pour enlever les warnings sur le mock
```suggestion
/* eslint-disable @typescript-eslint/no-unused-vars */
import { UsersRepository } from '../../repositories/users-repository';
```

Reply:
Extra merci ! :D